### ISSUE TITLE
Unable to login to test environment with production credentials

### ISSUE THREAD

**Issue by RobbS79** [allegro/allegro-api#9937]
Hello there!

<img width="417" alt="Snímek obrazovky 2024-09-20 v 11 56 38" src="https://github.com/user-attachments/assets/6674dacb-e456-4c9f-8f20-c5aab55a641f">

1. After clicking, it redirects me to the - allegro.pl.allegrosandbox.pl
2. Then after input of account credentials I receive a message - Captcha is required, however it does not provide captcha approval

<img width="507" alt="Snímek obrazovky 2024-09-20 v 11 58 49" src="https://github.com/user-attachments/assets/0ebbd3c3-e5c7-4142-814e-a2ad94c02ce3">

I have been trying to register an application for over 6 hours in sandbox environment. Your colleague verified my account this morning, for while I could get into apps.developer.allegro.pl.sandbox.pl (or simmilar) with orange-active REGISTER APPLICATION button, but when I filled the fields and clicked on final REGISTER button, there was no response. Please provide neccessarry steps, I am quite sure I progress according to the documentation, but without any further profgress. Thank you!

**Comment by PrzemyslawLukanowski** [comment]
Also, now I see that you've written "Unable to login to test environment with production credentials" - you need to use your sandbox password, it is a separate environment from production.

**Comment by RobbS79** [comment]
How about email? Can email be a real, the one which I use for production account? Nothing seems to work. 
1. I am attempting to recover password with production email address.
output: 
<img width="627" alt="Snímek obrazovky 2024-09-20 v 13 06 10" src="https://github.com/user-attachments/assets/873fe0c7-c242-4ae6-a211-ff7fcc20d439">

2. Create a new account with new email address on allegro.pl.allegrosandbox.pl
output:
<img width="615" alt="Snímek obrazovky 2024-09-20 v 13 10 06" src="https://github.com/user-attachments/assets/f7a54fc5-f13f-4a82-a36f-3b0cce0eabbb">

**Comment by PrzemyslawLukanowski** [comment]
Yes, the email needs to be real and it can be the same as on production. Could you please remove cookies or try to recover the password in incognito mode? If the error still occurs, please attach a screenshot of what the console returns.

**Comment by RobbS79** [comment]
Problem remains:
<img width="1056" alt="Snímek obrazovky 2024-09-20 v 13 26 53" src="https://github.com/user-attachments/assets/2cdea187-4367-4d71-8e84-430e9205b277">

**Comment by RobbS79** [comment]
Moreover, in incognito mode, now it does not even allow me to verify Captcha:
![Uploading Snímek obrazovky 2024-09-20 v 13.30.59.png…]()

**Comment by PrzemyslawLukanowski** [comment]
Thank you, I've forwarded this issue to the team responsible for this functionality. Sorry for the inconvenience.

**Comment by RobbS79** [comment]
Thank you! Please, let me know when you get updated.

**Comment by PrzemyslawLukanowski** [comment]
Hi, @RobbS79 Could you please try to log in once again? We verified our logs but didn't find anything suspicious and we also could not recreate this issue.
